Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
+------------------
+
+.. data:: CLOCK_HIGHRES
+
+   The Solaris OS has a CLOCK_HIGHRES timer that attempts to use an optimal
```

Comment:
Yeah, my comment wasn't clear. I meant to say replace CLOCK_HIGHRES with \`\`CLOCK_HIGHRES\`\`.